app/res/navigation/nav_graph_connectid.xml

```diff
@@ -165,6 +165,10 @@
             android:name="password"
             app:argType="string"
             app:nullable="true" />
+        <argument
+            android:name="isDismissible"
```

Comment:
lets just call it  and all other relevant variables `isCancellable` to be consistent with `setCancellable` in the message fragment.